animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/FeedingPointDetailsViewModel.swift

```diff
@@ -13,6 +14,16 @@ final class FeedingPointDetailsViewModel: FeedingPointDetailsViewModelLifeCycle,
     var onContentHaveBeenPrepared: ((FeedingPointDetailsViewMapper.FeedingPointDetailsViewItem) -> Void)?
     var onMediaContentHaveBeenPrepared: ((FeedingPointDetailsViewMapper.FeedingPointMediaContent) -> Void)?
     var onFavoriteMutationFailed: (() -> Void)?
+    
+    var showOnMapAction: ButtonView.Model? {
```

Comment:
I hope I've found the correct place for action
@diantyn

Reply:
nit: as for me the best place to pass another action is `content: FeedingPointDetailsViewMapper.FeedingPointDetailsViewItem` due to fact that this struct already contains field with action.